pages/studentPortal/intro/React/1.tsx

```diff
@@ -53,6 +53,7 @@ const React1 = () => {
               <div className="grid grid-cols-1 gap-4 sm:grid-cols-2">
                 {data.map((it) => (
                   <Card
+                    color="red"
```

Comment:
Why red?

Reply:
I don't see red in your screenshot

Reply:
ah - i changed this to something random because it looks like you added a color prop to the Card component so the compiler was failing... removed this change but will need to be fixed eventually